src/internal/connector/graph/uploadsession.go

```diff
@@ -29,11 +29,11 @@ type writer struct {
 	contentLength int64
 	// Last item offset that was written to
 	lastWrittenOffset int64
-	client            *resty.Client
+	client            httpWrapper
 }
 
 func NewWriter(id, url string, size int64) *writer {
```

Comment:
The very generic names here concern me more after moving this file to the graph package.  When it was in `uploadsession`, the scope of the writer was self evident by the package name.  But a `graph.writer` could mean a lot of things, whereas this has a very specific purpose.  How about a name like `LargeItemWriter`?

Reply:
Agreed. I have done the changes.